app/Services/AuthServiceProvider.js

```diff
@@ -101,7 +112,7 @@ async function destroyUser(userId) {
     },
   });
   if (counter === 0) {
-    throw new ApiError(httpStatus.NOT_FOUND, 'Account not found');
+    throw new ApiError(httpStatus.NOT_FOUND, 'Account not found', 'not-found');
```

Comment:
I think there is a fields property unnecessary.

Reply:
resolved